funcx_endpoint/tests/funcx_endpoint/executors/high_throughput/test_funcx_worker.py

```diff
@@ -0,0 +1,25 @@
+from funcx_endpoint.executors.high_throughput.funcx_worker import FuncXWorker
+from funcx_endpoint.executors.high_throughput.messages import Task
+import os
+import pickle
+
+
+class TestWorker:
+    def test_register_and_kill(self, mocker):
+        mocker.patch('funcx_endpoint.executors.high_throughput.funcx_worker.sys.exit')
```

Comment:
Add a comment here to explain the sequence of steps, and a note on why sys.exit is mocked.